guides/05_custom-components/05_frontend.md

```diff
@@ -0,0 +1,227 @@
+# The Frontend 🌐⭐️
+
+This guide will cover everything you need to know to implement your custom component's front-end.
+
+>💡 Tip: Gradio components use svelte. Writing svelte is fun! If you're not familiar, check out the interactive [guide](https://learn.svelte.dev/tutorial/welcome-to-svelte)
+
+## The directory structure 
+
+The frontend code should have at the minimum three files:
+
+* `Index.svelte`: This is the main export and where your component's layout and logic should live.
+* `Example.svelte`: This is where the example view of the component is defined.
+
+Feel free to add additional files and subdirectories. 
+If you want to export any additional modules, remember to modify the `package.json` file
+
+```json
+"exports": {
+    ".": "./Index.svelte",
+    "./example": "./Example.svelte",
+    "./package.json": "./package.json"
+},
+```
+
+## The Index.svelte file
+
+Your component should expose the following props that will be passed down from the parent Gradio application.
+
+```
```

Comment:
Fixed this, now you can use javascript or js and it will render correctly: 

![Screen Shot 2023-10-29 at 10 35 32 PM](https://github.com/gradio-app/gradio/assets/9021060/1766de24-b921-47b3-84a8-6f1ca709b936)

Although you should use ts here and it will highlight the word type too

Reply:
I didn't change the rest of the codeblocks though if there are more